### PR TITLE
Use Glue enrollee individual premiums instead of calculation using plan rate table in Coverage information Serializer

### PR DESCRIPTION
**Pull request by saipraveen18**
ME-181218888

### CODE REVIEW

File: app/models/generators/coverage_information_serializer.rb [hunk 1]
```diff
@@ -116,17 +117,48 @@ def append_financial_information(enrollee, financial_dates=nil)
...
+                          aptc_credit.aptc.to_f
+                        else
+                          ehb_amount = as_dollars(total_premium_amount * @policy.plan.ehb)
+                          @policy.applied_aptc.to_f > ehb_amount.to_f ? ehb_amount : @policy.applied_aptc.to_f
+                        end
+
+          total_responsible_amount = if aptc_credit.present?
+                                       aptc_credit.tot_res_amt.to_f
+                                     else
+                                       as_dollars((total_premium_amount - aptc_amount)).to_f
```
Comment (nks2109) [inline app/models/generators/coverage_information_serializer.rb:136]: you can do this as_dollars(total_premium_amount - aptc_amount)

**Review by nks2109 — CHANGES_REQUESTED** [review]

File: app/models/generators/coverage_information_serializer.rb [hunk 2]
```diff
@@ -7,7 +7,9 @@ class CoverageInformationSerializer
     def initialize(person, plan_ids = nil)
       @person = person
       if plan_ids
-        @policies = person.policies.where({"plan_id" => {"$in" => plan_ids}})
+        @policies = Policy.where({:enrollees => {"$elemMatch" => {:rel_code => "self",
+                                                                  :m_id => person.authority_member_id}},
```
[the remark this answers is not in the document]
Comment (saipraveen18) [reply app/models/generators/coverage_information_serializer.rb:11]: Not sure  - why can't we take authority member id on person? Does enrollee m_id and person's authority_member_id not the same??

File: app/models/generators/coverage_information_serializer.rb (hunk 1 above)
Comment (saipraveen18) [reply app/models/generators/coverage_information_serializer.rb:136]: I am using as_dollars and also to_f, will this cause any issues?

File: app/models/generators/coverage_information_serializer.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (nks2109) [reply app/models/generators/coverage_information_serializer.rb:11]: In some cases, we might just put the value of authority_member_id on a person object but the actual m_id is the authority_member's hbx_member_id.

**Review by nks2109 — APPROVED** [review]
Premium values are not considering from plan table which looks good.

**Review by nks2109 — APPROVED** [review]

File: app/models/generators/coverage_information_serializer.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (saipraveen18) [reply app/models/generators/coverage_information_serializer.rb]: Updated. Fetching m_id from person's authority member hbx_member_id

**Pull request merged**